shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.ivy.ui
+
+import android.content.Context
+import androidx.test.platform.app.InstrumentationRegistry
+import com.ivy.domain.features.BoolFeature
+import com.ivy.domain.features.FeatureGroup
+import com.ivy.domain.features.Features
+import com.ivy.ui.time.DevicePreferences
+import io.kotest.common.runBlocking
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+import java.util.Locale
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class FormatMoneyUseCaseTest(private val param: TestData) {
+
+    private val context: Context = InstrumentationRegistry.getInstrumentation().context
```

Comment:
@shamim-emon you don't need Roboelectric. Just mock the context 
```
every { features.showDecimal } returns mockk { every { isEnabled(any()) } returns true }
```

Reply:
Done